packages/@react-spectrum/label/src/Field.tsx

```diff
@@ -79,12 +79,23 @@ function Field(props: SpectrumFieldProps, ref: RefObject<HTMLElement>) {
         showErrorIcon={showErrorIcon} />
     );
 
-    let renderChildren = () => (
-      <Flex direction="column" UNSAFE_className={classNames(labelStyles, 'spectrum-Field-wrapper')}>
-        {children}
-        {hasHelpText && renderHelpText()}
-      </Flex>
-    );
+    let renderChildren = () => {
+      if (labelPosition === 'side') {
```

Comment:
why did this block of code change? label position side has been working so far?

Reply:
if it's related to the test for Field below, I had a bunch of tests I was suggesting to Yihui that may be of interest

https://github.com/adobe/react-spectrum/pull/3320#pullrequestreview-1057973922

obviously you don't need to differentiate with readonly like I was in those

Reply:
This was from Michael's original PR. I think the extra wrapper isn't needed when the label is on top

Reply:
Yeah, without this change, the textarea doesn't expand to the full height.